docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -262,9 +262,10 @@ that import the namespaced Kubernetes resources (kinds of objects) for managemen
 containerized workloads. At the completion of `kubestellar init` the current workspace will be
 "root".
 
-6. Creates the space provider objects to be used by the space manager. Need to pass
-in two environment variables - SM_CONFIG and IN_CLUSTER. SM_CONFIG
-is the path to the kubeconfig for the space manager. IN_CLUSTER specify whether spaces
+6. Creates the space provider objects to be used by the space manager. It requires
+several environment variables - SM_CONFIG, SM_CONTEXT, and IN_CLUSTER. SM_CONFIG
+is the path to the kubeconfig for the space manager. The SM_CONTEXT is the space manager
+context in the config file. The IN_CLUSTER variable specify whether spaces
 are accessed from within the hosting cluster or externally by kubestellar init.
```

Comment:
This is not about _all_ access, it it specifically about the accesses done by this command. Please clarify the text.